liquidity_pool/src/utils.rs

```diff
@@ -66,6 +80,16 @@ pub trait UtilsModule:
             .unwrap()
     }
 
+    fn get_token_ticker_from_lending(&self, token_id: &TokenIdentifier) -> ManagedBuffer {
+        let as_buffer = token_id.as_managed_buffer();
+        let ticker_start_index = 1;
```

Comment:
Maybe add a comment here, something like "Each lend token has an L prefix, so we start from index 1". This could easily be considered an indexing mistake otherwise.